cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -379,6 +380,9 @@ private class ServiceImpl(
                     )
                     sPConsentsSuccess?.invoke(consentHandler)
                 }
+                .executeOnLeft { error ->
+                    (error as? ConsentLibExceptionK)?.let { logger.error(error) }
```

Comment:
@bohdan-go-wombat will the error be `ConsentLibExceptionK` 100% of the time? What will happen if it won't?

Reply:
it will be 100% `ConsentLibExceptionK`, since we have a mechanism to transform any exception to the type of `ConsentLibExceptionK`